hugo.yaml

```
@@ -0,0 +1,189 @@
+baseURL: "https://open-cluster-management.io"
+title: "Open Cluster Management"
+
+# Language settings
+contentDir: content/en
+defaultContentLanguage: en
+defaultContentLanguageInSubdir: false
+# Useful when translating.
+enableMissingTranslationPlaceholders: true
+
+enableRobotsTXT: true
+
+# Will give values to .Lastmod etc.
+enableGitInfo: true
+
+# Comment out to enable taxonomies in Docsy
+# disableKinds: [taxonomy, taxonomyTerm]
+
+# You can add your own taxonomies
+taxonomies:
+  tag: tags
+  category: categories
+
+# Highlighting config
+pygmentsCodeFences: true
+pygmentsUseClasses: false
+# Use the new Chroma Go highlighter in Hugo.
+pygmentsUseClassic: false
+# pygmentsOptions: "linenos=table"
+# See https://help.farbox.com/pygments.html
+pygmentsStyle: tango
+
+# Configure how URLs look like per section.
+permalinks:
+  blog: /:section/:year/:month/:day/:slug/
```

This results in an oddly long URL with a lot of slashes. Since they're only subdivided by year, what about:

```suggestion
  blog: /:section/:year/:slug/
```

Or use the date with hypens?
```
  blog: /:section/:year-:month:-:day/:slug/
```

---

Updated.